### ISSUE TITLE
Can I insert A, B test data in order at test mode?

### ISSUE THREAD

**Issue by khryang** [junyanz/pytorch-CycleGAN-and-pix2pix#101]
at test mode, the data of testB is inserted randomly.
for that reason, the B test set is not transform whole.
I want to transform the whole data of B set as well as test A set.
is there any fault in my process?
how can I do? please any comment.

**Comment by khryang** [comment]
I did.
my 'unaligned dataset.py' file was not updated.
now I can do, thanks!

**Comment by wlwsea** [comment]
Hello @khryang ,  
I have the same problem with you.
I just used the command as follows, but the fakeA data i got is randomly.
`python train.py --dataroot ./datasets/maps --name maps_cyclegan --model cycle_gan --no_dropout`
I have the file 'unaligned dataset.py' , but i don't know how to use it.
Could you tell me some details about how you did it ?
Thanks a ton !

**Comment by khryang** [comment]
@wlwsea  I changed the code how data insert in 'unaligned_dataset.py'.
look at the part of 'def __getitem__' .  the index of B is random.
At the test mode, I use " index_B = index_A" instead of random code.

**Comment by wlwsea** [comment]
Hi  @khryang ,
It really work !
Thank you !
